sklearn/cluster/tests/test_k_means.py

```diff
@@ -332,9 +332,10 @@ def test_k_means_fit_predict(algo, dtype, constructor, seed, max_iter, tol):
     # There's a very small chance of failure with elkan on unstructured dataset
     # because predict method uses fast euclidean distances computation which
     # may cause small numerical instabilities.
-    # NB: This test is largely redundant wrt test_predict and test_predict_equal_labels
-    #     This has the added effect of testing idempotence of the fittng procesdure
-    #     which appears to be where it fails on some MacOS setups.
+    # NB: This test is largely redundant with respect to test_predict and
+    #     test_predict_equal_labels.  This test has the added effect of
+    #     testing idempotence of the fittng procesdure which appears to
```

Comment:
typo: fittng procesdure but fair enough :)

Reply:
I'm still not used to the keyboard on these new macbooks